float32 X
float32 Y
float32 Z
float32 distance
float32 heading
float32 pitch
float32 roll
float32 velX
float32 velY
float32 velZ
float32 velP
float32 velH
float64 latitude
float64 longitude
float32 altitude
int8 ValidGPSfix
float32 X_dead
float32 Y_dead
float32 velX_dead
float32 velY_dead
float32 temperature
int32 frame0
int32 frame1
float32 velP_dead
